### DISCUSSION TITLE
Using Uno Wasm for Cloudflare Workers

### DISCUSSION THREAD

**Discussion by Flori00123** [unoplatform/Uno.Wasm.Bootstrap#469] [Ideas]
Hey,

i wanted to know if it would be possible with your libary to make the c# code to wasm and host it on cloudflare workers? I already tried it and came across some problems. When i use python for example, it generates a addEventListener and handleRequest function in javascript, but i can write them per cleartext in the index.py without errors, does anyone know how to do that in your libary?

I rly think serverless computing and wasm are cool things to work with, but unfortunately i can't find anything how to operate them in c#. These are all currently supported lang for cloudflare workers: https://developers.cloudflare.com/workers/platform/languages

`
def handleRequest(request):
    return __new__(Response('Python Worker hello world!', {
        'headers' : { 'content-type' : 'text/plain' }
    }))

addEventListener('fetch', (lambda event: event.respondWith(handleRequest(event.request))))
`

Thanks!

**Comment by jeromelaban** [comment]
Thanks for your interest! We haven't investigated this yet, but seeing that https://github.com/cloudflare/worker-emscripten-template is available, there's probably a way to get this running somehow.

That being said, the .NET team is looking at getting the runtime running under wasi, which would make any javascript dependency go away, and make it suitable for running better in workers. Maybe for .NET 7!
